code/sound/sound.cpp

```diff
@@ -655,20 +655,26 @@ sound_handle snd_play_3d(game_snd* gs, vec3d* source_pos, vec3d* listen_pos, flo
 		Int3();
 		return sound_handle::invalid();
 	}
+	if (gs->flags & GAME_SND_NOT_VALID) {
```

Comment:
This check should be added to `snd_play()` and `snd_play_looping()` as well, correct?

I assume Issue #3388 is not limited to 3D sounds?

Reply:
Good point.  I don't know if #3388 is limited to 3D sounds or not, but it makes sense that other sounds would behave similarly.